docs/source/triggers/template-root/daml.yaml.template

```diff
@@ -11,3 +11,5 @@ dependencies:
   - daml-stdlib
   - daml-trigger
 # trigger-dependencies-end
+sandbox-options:
```

Comment:
The docs for triggers and DAML script currently tell you to pass `--static-time` to `daml script` and `daml trigger` (I switched them over to requiring the time to be specified explicitly a while back already). With this change that is inconsistent. There is no reason for why they need static-time, I simply didn’t want to deviate from the default so the best fix is to use --wall-clock-time in the docs as well.

Reply:
OK, but even if you're using `daml start`? If you use `daml sandbox`, you need to use `--wall-clock-time`.

Reply:
This is about the argument passed to the script runner and the trigger runner. With this change running `daml start` and the starting the trigger runner with `daml trigger --static-time` is definitely not going to work. So all that’s necessary is to also flip the mode in the invocations to `daml trigger` and `daml script`.

Reply:
Done.